docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -410,6 +420,34 @@ The `getYearRange` method used to accept two params and now accepts a tuple to b
 + adapter.getYearRange([start, end])
 ```
 
+### Restrict the input format of the `date` method
+
+The `date` method now have the behavior of the v6 `dateWithTimezone` method.
+The following changes have been applied:
+
+- Its first argument is now mandatory, you need to pass `undefined` to get a date with the current timestamp instead of passing no argument
+
+  ```diff
+  - adapter.date();
+  + adapter.date();
+  ```
+
+- It no longer accept `any` as a value but only `string | null | undefined`
+
+  ```diff
+  - adapter.date(new Date());
+  + adapter.date();
+
+  - adapter.date(new Date('2022-04-17');
+  + adapter.date('2022-04-17');
+
+  - adapter.date(new Date(2022, 3, 17));
+  + adapter.date('2022-04-17');
```

Comment:
Wondering if one of those two examples should have time section, such that it's clear that `.date()` supports hours, minues, ...